clients/java/src/main/java/io/zeebe/client/impl/ZeebeClientBuilderImpl.java

```diff
@@ -221,6 +245,31 @@ public ZeebeClient build() {
     return new ZeebeClientImpl(this);
   }
 
+  private ZeebeClientBuilder keepAlive(final String keepAlive) {
+    if (keepAlive == null || keepAlive.length() < 2) {
+      throw new IllegalArgumentException(String.format(ILLEGAL_KEEP_ALIVE_FMT, keepAlive));
+    }
+
+    final int timeValue;
+    try {
+      timeValue = Integer.parseUnsignedInt(keepAlive.substring(0, keepAlive.length() - 1));
```

Comment:
I think we have a `DurationUtil`, though not sure if its available as a dependency here? Otherwise I would say let's keep things simple and just force people to put the keep alive as MS? What do you think?

Reply:
Didn't know about DurationUtil. Fits perfectly here, thanks